pages/windows/get-content.md

```diff
@@ -0,0 +1,16 @@
+# Get-Content
+
+> Get the content of the item at the specified location.
+> More information: <https://docs.microsoft.com/powershell/module/microsoft.powershell.management/get-content>.
+
+- Get the content of a file:
+
+`Get-Content -Path {{file_name}}`
+
+- Limit the number of lines returned by Get-Content:
+
+`Get-Content -Path {{file_name}} -TotalCount {{number_of_lines}}`
+
+- Get the content of the file and keeps it open to print each new line:
```

Comment:
We can't use `keeps` because we shouldn't use `present simple`. I think this is better:
```suggestion
- Display the content of the file and keep reading it until `Ctrl + C`:
```

Reply:
What about this, @navarroaxel?

```suggestion
- Display the content of the file and keep reading from it until `Ctrl + C` is pressed:
```